src/app/app.component.html

```diff
@@ -1,2 +1,9 @@
 <router-outlet></router-outlet>
-<h2>Hello World</h2>
+<h2>OpenBabel Web UI</h2>
+<mat-card>
+  <mat-card-content>
+    <app-input class="component"></app-input>
```

Comment:
Is there a reason/need to have same class name for all the components? If not, can you give each one a distinctive class name e.g "class="options-input-component"